site/upgrading-to-1.0.md

```diff
@@ -0,0 +1,257 @@
+    <li>For other changes, commit them to your git repo</li>
+    <li>Flux applies all changes to the git repo to the cluster</li>
+   </ul>
+  </td>
+ </tr>
+</table>
+
+## Upgrade process
+
+In summary, you will need to:
+
+ 1. Remove the old flux resources from your cluster
+ 2. Delete any deployment keys 
+ 3. Run the new flux resources
+ 4. Install a new deploy key
+
+First, it will help in a few places to have an old fluxctl
+around. Download it from GitHub:
+
+    curl -o fluxctl_030 https://github.com/weaveworks/flux/releases/download/0.3.0/fluxctl_linux_amd64
+    # or if using MaxOSX,
+    # curl -o fluxctl_030 https://github.com/weaveworks/flux/releases/download/0.3.0/fluxctl_darwin_amd64
+    chmod a+x ./fluxctl_030
+
+The next steps depend on whether you
+
+ * [Connect Flux to Weave Cloud](#if-you-are-connecting-to-weave-cloud); or,
+ * [You are running Flux standalone](#if-you-are-running-flux-in-standalone-mode) (i.e., you run both
+   the Flux daemon and the Flux service, rather than going through
+   Weave Cloud).
+
+## <a name="weavecloud">If you are connecting to Weave Cloud</a>
+
+Set the environment variable `FLUX_SERVICE_TOKEN` to be your service
+token (found in the Weave Cloud UI for the instance you are
+upgrading).
+
+Before making changes, get the config so that it can be consulted
+later:
+
+    ./fluxctl_030 get-config --fingerprint=md5 | tee old-config.yaml
+
+### Remove old Flux resources
+
+> *Important! If you have Flux resources committed to git*
+> 
+> The first thing to do is remove any manifests for running Flux that
+> you have stored in git. Otherwise, when the new Flux daemon runs it
+> will restore the old configuration.
+
+Run Weave Cloud’s launch generator to delete the resources:
+
+    kubectl delete -n kube-system -f \
+      https://cloud.weave.works/k8s/flux.yaml?flux-version=0.3.0”
+
+### Delete deployment keys
+
+It’s good practice to remove any unused deployment keys. If you’re
+using GitHub, look at the settings for the repository you were
+pointing Flux at, and delete the key Flux was using.
+
+To check you are removing the correct key, get the fingerprint of the
+key used by Flux with
+
+    ./fluxctl_030 get-config --fingerprint=md5
+
+### Configure and run the new flux resources
+
+> First, it is important to understand that Flux manages more of your
+> cluster resources now. It will automatically apply manifests that
+> appear in your config repo, either by creating or by updating them. In
+> other words, it tries to keep the cluster running whatever is
+> represented in the repo. (Though it doesn’t delete things, yet.)
+
+To run the new Flux with Weave Cloud: 
+
+ * Go to your instance settings (the cog icon) and click “Deploy Config”
+ * Enter the git URL, path and branch values from the saved config
+   (old-config.yaml)
+ * Run the `kubectl` command shown below the form.
+ * Wait for the 
```

Comment:
something missing?

Reply:
Oops. Done in a hurry is my only defense.

Reply:
Oh yes: I write these in as stubs because I went away to check what you had to wait for, and didn't come back.